managed/ui/src/redesign/helpers/api.ts

```diff
@@ -112,6 +113,16 @@ class ApiService {
   isRequestCancelError(error: unknown): boolean {
     return axios.isCancel(error);
   }
+
+  /**
+   * Delete certificate which is not attched to any universe.
+   * 
+   * @param certUUID - certificate UUID
+   */
+  deleteCertificate = (certUUID: string): Promise<any> => {
```

Comment:
if we don't expect any response from the delete API call - the return type should be `Promise<void>`

Reply:
Done